tensornetwork/tensornetwork_test.py

```diff
@@ -426,6 +426,12 @@ def test_outer_product(backend):
   np.testing.assert_allclose(d.tensor, np.ones((2, 4, 5, 4, 3, 6)))
   assert d.name == "D"
 
+  # check outer product with scalar tensor
```

Comment:
Can you make this a separate test? If the earlier part of the test fails, this part won't run.